docs/application/dotnet/samples/watch/overview.md

```diff
@@ -1,23 +1,26 @@
 # Watch
 
+(M) : for mobile profile, (W) : for wearable profile, (T) : for TV profile
 <table>
 	<tbody>
 		<tr>
 			<td><img alt="" height="180" src="media/wambientwatch.png" width="180"/></td>
 			<td>
 			<p><strong>(W) AmbientWatch</strong> [In progress]</p>
-			<p>The AmbientWatch application demonstrates how to make a watch application that supports ambient mode in Tizen wearable device.
-			To extend battery life, a watch application can display a limited UI in ambient mode by detecting ambient mode changes.<br>
+			<p>The AmbientWatch application demonstrates how to create a watch application that supports ambient mode in Tizen wearable device.
+			To extend battery life, a watch application can display a limited UI in the ambient mode by detecting ambient mode changes.<br>
 			</p>
-			<p>First of all, <i>"ambient-support"</i> attribute should be set to true in <b>tizen-manifest.xml</b> file as follows:<br>
+			<p>In <b>tizen-manifest.xml</b> file, set <code>"ambient-support"</code> attribute to true:<br>
+			For more information, see ambient-support attribute.
+			<br>
 			<code>
 			&lt;watch-application appid&equals;&quot;org.tizen.example.AmbientWatch&quot;  exec&equals;&quot;AmbientWatch.dll&quot; type&equals;&quot;dotnet&quot;  ambient-support&equals;&quot;true&quot;&gt;</code>
 			</p>
 			<p>
 			You can get details about <a href="https://developer.tizen.org/development/tizen-studio/native-tools/configuring-your-app/manifest-text-editor" target="_blank">ambient-support attribute</a>.
 			</p>
 			<p>
-			To use the ambient mode, you must enable it in Settings application. Launch "Settings" App -> Select "Watch faces and styles" -> Choose "Watch always on" -> Enable it. After screen timeout, a wearable device turns into ambient mode. In addition, the ambient mode activates only when you are wearing a watch on your wrist.</p>
+			To use the ambient mode, you must enable it in the application settings. Launch <b>Settings</b> App -> <b>Watch faces and styles</b> -> <b>Watch always on</b> -> enable it. After screen timeout, the wearable device turns into ambient mode. In addition, the ambient mode activates only if you are wearing the watch on your wrist.</p>
```

Comment:
Launch **Settings** App > **Watch faces and styles**.......
Here "App" is needed?
Or can we rephrase as:
Launch **Settings** in the application....?

Reply:
done

Reply:
In #331,
the style used:  **Tools** > **Tizen** > **Tizen Device Manager**.
To maintain consistency, can we use the same style ">" instead of "->"?

If there is any specific reason to use this style, then ignore my comments.

Reply:
done